js/page-specific/phone-directory/phone-directory.html

```diff
@@ -1,245 +1,125 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 
 <html xmlns="http://www.w3.org/1999/xhtml">
-  <head id="Head1">
+<head id="Head1">
     <title>
-      Baltimore County Employee Phone Directory
+        Baltimore County Employee Phone Directory
     </title>
-    <script
-      src="https://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"
-      type="text/javascript"
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"
+            type="text/javascript"
     ></script>
-    <!-- page head text -->
-
-    <script type="text/javascript">
-      var jQuery = $.noConflict(true);
-    </script>
-
-    <script
-      type="text/javascript"
-      src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-1.5.1.min.js"
+    <script type="text/javascript"
```

Comment:
Not blocking but we may want to create a note about this path, once we move the js file.

Reply:
What are you thinking - maybe an inline comment noting that this version of jquery was removed, in case we need it again for some reason?

Reply:
More that once we upload `phone-directory.js` to our cms, the path would change depending on the environment.

Reply:
also not blocking.